Yoast/wordpress-seo
# Remove Language Notice code

## Context


* This code has been disabled by passing a hard "false" in https://github.com/Yoast/wordpress-seo/issues/11034 it just needed to be cleaned up.

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Use the metabox and see everythin working as expected
* This code has been disabled for a while; removing the code should not break anything

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/javascript/issues/289
